docs/exact.md

```diff
@@ -18,6 +18,7 @@ Second, it allows us to optimize the case when we have multiple representations
 
 Then suppose we have an expression `v[i]` where the static type of `v` is `int[]`.  We can compile this into code that checks the exact bit, and if it is set, performs the access knowing that representation 2 is being used, and otherwise uses a `get` function pointer accessed via the list's descriptor. When there are only 3 representations, this could be done easily without exact tracking. But with exact tracking we can efficiently have many optimized representations, for example a representation for every tuple type: if we have a type `[int,int]` we can represent it as an LLVM `{i64*, i64, i64}`, where the initial `i64*` points to a list descriptor. This is particularly useful for records: we can use a representation similar to what would be used in a language like C++.
 
+Finally, it allows us to check if the compile time type of the function variable is the same as the inherent type. If so we can avoid calling the function via the uniform function and instead perform an indirect jump using the executable code pointer.
```

Comment:
compile time type of the function reference

Reply:
Fixed with #1240